src/deepsparse/server/server.py

```diff
@@ -242,8 +245,9 @@ def _add_pipeline_endpoint(
 
     def _predict(request: pipeline.input_schema):
         pipeline_outputs = pipeline(request)
-        if pipeline.logger:
-            log_resource_utilization(pipeline)
+        server_logger = pipeline.logger
+        if server_logger:
```

Comment:
how would a user disable this? shouldn't this be controlled by the config?

Reply:
Good point. After removing the SystemLoggingManager object I did neglect this detail.

Reply:
I probably have made an erroneous implicit assumption. When we decided early on to move from an object that will control all sorts of system logging activities (I called it `SystemLoggingManager`) in favor of separate, simple functions, I explicitly assumed that this would also mean that for now the only instance that enables/disables the logging is the presence or absence of the appropriate function logger. Not the most scalable solution, but sufficient for now. Let's sync on that as well.